examples/julia_logo.jl

```diff
@@ -0,0 +1,159 @@
+
+    line(p1, p2, :stroke)                   # vector line
+    poly([p2, v1, v2, p2], :fill)           # vector tip
+end
+
+function draw_logo(n::Int; frames = 2160)
+    # TODO: (1) Fix indices so that I don't need to go to end-1 (a problem for very small n)
+
+    base_size = 7.5e-5
+    n_rotations = 2         # how many times the "tip" goes around the drawing
+
+    remap_idx(i::Int) = (-1)^(i - 1) * floor(Int, (i + 1) / 2)
+    remap_inv(n::Int) = 2n * sign(n) - 1 * (n > 0)
+
+    n0, nT = -Int((n - 1) / 2), Int((n - 1) / 2)
+    ids = OffsetArray(map(i -> Symbol("ball_" * "$(i)"), n0:nT), n0:nT)
+    indices = [remap_idx(i) for i in 0:(n - 1)]
+
+    fade_fourier() = [
+        SubAction(1:floor(Int, frames / 8), appear(:fade)),
+        SubAction(floor(Int, 0.5 * frames):floor(Int, 0.9 * frames), disappear(:fade)),
+    ]
+    fade_main() = [
+        SubAction(1:floor(Int, frames / 20), appear(:fade)),
+        SubAction(floor(Int, 0.9 * frames):frames, disappear(:fade)),
+    ]
+
+    yellow_path = Point[]
+    javis(
+        vid,
+        [
+            BackgroundAction(1:frames, ground),
+            Action(
+                1:frames,
+                ids[0],
+                (args...) ->
+                    object(Point(base_size * cs[0]), size = abs(base_size * cs[1])),
+                Rotation(0.0, 0.0),
+                subactions = fade_fourier(),
+            ),
+            map(
+                n -> Action(
+                    1:frames,
+                    (args...) -> vector(
+                        pos(ids[indices[remap_inv(n)]]),
+                        pos(ids[indices[remap_inv(n) + 1]]),
+                        opacity = 0.9,
+                        linewidth = 2,
+                    ),
+                    subactions = fade_fourier(),
+                ),
+                indices[2:(end - 1)],
+            )...,
+            map(
+                n -> Action(
+                    1:frames,
+                    ids[n],
+                    (args...) -> object(
+                        Point(base_size * cs[n]),
+                        size = abs(base_size * cs[indices[remap_inv(n) + 2]]),
+                    ),
+                    Rotation(0.0, n_rotations * 2π * n, ids[indices[remap_inv(n)]]),
+                    subactions = fade_fourier(),
+                ),
+                indices[2:(end - 1)],
+            )...,
+            Action(
+                1:frames,
+                (args...) -> draw_path!(yellow_path, pos(ids[indices[end - 1]]), "yellow");
+                subactions = fade_main(),
+            ),
+            Action(
+                1:frames,
+                :tip,
+                (args...) -> object(
+                    Point(base_size * cs[indices[end - 1]]),
+                    "yellow",
+                    size = 4,
+                    style = :fill,
+                    opacity = 1.0,
+                ),
+                Rotation(0.0, 0.0, ids[indices[end - 1]]),
```

Comment:
feels a bit odd to have a non existing rotation here but I don't really have a better way to express this :smile:

Reply:
Yes, I just needed the "tip" to track the last point but didn't know how else to do it

Reply:
It's basically a translation, right? But yeah this works fine.

Reply:
Ah, I haven't worked with translation yet. Should I just use `Translation(p)` where `p` is the point I want to track? Because it does not really have `from` and `to` points

Reply:
I think `Translation(p,p)` should do it but yeah it's not more readable. We actually think about this in #179 like defining the movement only inside the SubAction such that the current action is more like an object description which is static instead of having a movement itself.

Reply:
Alternatively I could define a convenience function `Track(p)` that does exactly the same thing (but that wouldn't really change things)

Reply:
Oh don't worry about this. It was more a not for myself to think about ways to make this stuff either. I like your hack it's just that I'm a bit embarrassed that one needs a hack for this 😂